maqs-appium/src/test/java/com/cognizantsoftvision/maqs/appium/MobileDriverManagerUnitTest.java

```diff
@@ -15,7 +13,7 @@
 /**
  * The type Mobile driver manager unit test.
  */
-public class MobileDriverManagerUnitTest extends BaseGenericTest {
+public class MobileDriverManagerUnitTest extends BaseAppiumTest {
```

Comment:
Why extend base Appium when generic works?

Reply:
reverted back to use baseGenericTest